pkg/arvo/sys/lull.hoon

```diff
@@ -1147,6 +1147,11 @@
   ::                                                    ::::
 ++  eyre  ^?
   |%
+  +$  cache-entry
+    $%
+      [%scry =path case=@ud]
+      [%payload =simple-payload:http]
+    ==
```

Comment:
@lukechampine Public responses are the primary use case.
@Fang- Can you suggest an easy path forward? And yes, we should support alternative desks.

Reply:
I think simply accepting an `auth=?` as part of the `%add-cache` (as also suggested in my [earlier proposal](https://gist.github.com/Fang-/1939a5b51fb2f821d960e05b74ad5f9f#authentication)) is easiest. You'd store is as part of the `cache-entry`, and eventually give that to the runtime so it, too, knows.

Perhaps:
```suggestion
  +$  cache-entry
    $:  auth=?
    $=  body
    $%  [%scry =path case=@ud]
        [%payload =simple-payload:http]
    ==  ==
```

Reply:
@Fang- It was initially implemented as a `beam`: https://github.com/urbit/urbit/pull/5927/commits/0d34b861df944cf78ad65976a6fb747c02868fe0

I don't recall the exact reason we switched. Perhaps because it was too general; it wouldn't make sense to point a cache at `da+now`, for example.